package.json

```diff
@@ -15,18 +15,20 @@
   },
   "dependencies": {
     "escape-string-regexp": "^1.0.3",
-    "extend": "^2.0.1",
-    "postcss": "^5.0.5"
+    "extend": "^2.0.1 || ^3.0.1",
+    "postcss": "^5.0.5 || ^6.0.8"
```

Comment:
of course, was just trying to keep it backwards compatible. the `#append` method has been around since way before `postcss 5.x`. will update